bin/kano-track-space

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+#
+# Copyright (C) 2018 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
+#
+# kano-os-loader
+#
+# We install this script via cmdline.txt "init" kernel parameter
```

Comment:
Hmmm.. cmdline or systemd service below? If cmdline, I am a bit wary on running a python script as the main `init` process. The slightest problem or exception would bring the whole system down with a kernel panic...

Reply:
Sorry Albert, that's a typo - it's because I copied the header from kano-os-loader and forgot to update the comment.